runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -2164,7 +2163,13 @@ TR::Register *J9::Power::TreeEvaluator::checkcastAndNULLCHKEvaluator(TR::Node *n
 
 TR::Register *J9::Power::TreeEvaluator::newObjectEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   if (cg->comp()->suppressAllocationInlining())
+   // If the helper symbol set on the node is TR_newValue, we are (expecting to be)
+   // dealing with a value type. Since we do not fully support value types yet, always
+   // call the JIT helper to do the allocation.
+   //
+   auto* comp = cg->comp();
```

Comment:
Can you please put the type of TR::Compilation rather than auto - doing so aids readability and ensures that if we need to refactor later we can support from the compiler type checker.

Reply:
Fixed.